runelite-client/src/main/java/net/runelite/client/plugins/skybox/SkyboxPlugin.java

```diff
@@ -91,6 +103,17 @@ public void onBeforeRender(BeforeRender r)
 			return;
 		}
 
+		if
+		(
+			config.overrideMode() == SkyOverrideMode.ALL ||
+			(config.overrideMode() == SkyOverrideMode.OVERWORLD && client.getLocalPlayer().getWorldLocation().getY() < 4200)
```

Comment:
It looks to me like `4160` marks the first tile beyond the northernmost water, and this should also be defined in some publicly-accessible location (maybe `Constants`?) for general use--in the past, other plugins/features have wanted a reference to this value.

Reply:
Yup, seems good, adding it as Constants.OVERWORLD_MAX_Y.